components/calendar/test/calendar.visual-diff.js

```diff
@@ -145,15 +145,19 @@ describe('d2l-calendar', () => {
 
 	describe('interaction', () => {
 		afterEach(async() => {
-			await page.reload();
+			await page.$eval(firstCalendarOfPage, async(calendar) => {
```

Comment:
The fetching of the files added some time to reloading and having the calendars ready for interaction. If I added `{waitUntil: ['networkidle0', 'load']}` into the args for `reload` then that fixes things, but that also slows down the tests so I rearranged a bit in order to remove the reloading.